### PR TITLE
Fixing select block, updating volto to latest and adding less support

### LINKED ISSUE #48: Select block does not work as expected
1) as admin
2) create a new page with a select form block
3) add three options separated by a newline
4) save the form
-> the select widget will show a single option with all three options in one line instead of three separate options

Edit-Page

### PR DESCRIPTION
**Pull request by iFlameing**
Fixes  https://github.com/kitconcept/volto-form-builder/issues/48

### CODE REVIEW

**Comment by tisto** [comment]
@iFlameing ci is failing.

**Comment by tisto** [comment]
@iFlameing ci is failing.

**Comment by iFlameing** [comment]
@tisto When I am trying to run the acceptance test locally I am getting this error. Do you have any idea how I can solve it?

Screenshot from 2020-12-31 23-51-45

**Comment by tisto** [comment]
@iFlameing we removed Volto sending fullobjects param by default. Maybe this is related? cc @sneridagh

**Comment by tisto** [comment]
@iFlameing pls fix the confict.

**Comment by iFlameing** [comment]
@tisto done

File: cypress/integration/form.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ context('Form Blocks Tests', () => {
     cy.waitForResourceToLoad('@breadcrumbs');
     cy.waitForResourceToLoad('@actions');
     cy.waitForResourceToLoad('@types');
-    cy.waitForResourceToLoad('my-page?fullobjects');
+    // cy.waitForResourceToLoad('my-page?fullobjects');
```
Comment (tisto) [inline cypress/integration/form.js:16]: @iFlameing what happens if you just remove the "?fullobjects" from the call?

**Pull request merged**